src/cmd/bump-versions/main.go

```diff
@@ -2,88 +2,38 @@ package main
 
 import (
 	"context"
-	"flag"
 	"log/slog"
 	"os"
+	"registry-stable/cmd/common"
 	"registry-stable/internal/github"
 	"registry-stable/internal/module"
 	"registry-stable/internal/provider"
 )
 
 func main() {
-	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
-	logger.Info("Starting version bump process for modules and providers")
+	cli := common.Parse()
```

Comment:
**(idea/nitpick)** WDYT about the package `config` with the function `Configure()` instead? Motivation: the `common` package does not only parse CLI arguments, but also configures the app (e.g. defines the logger) following the builder pattern.

Reply:
I agree, I really don't like packages called things like "utils" or "common"

`config` would be good I think.

Reply:
Also it should go into `internal` instead of `cmd`, otherwise people will try and run `go run ./cmd/common`

Reply:
> I agree, I really don't like packages called things like "utils" or "common"

I'd be a proponent of the team agreement to _not_ use those generic names as they have the "hide under the carpet" effect and carry not much meaning.